src/App.js

```diff
@@ -14,6 +14,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Books />} />
         <Route path="/Category" element={<Category />} />
+        <Route path="/Form" element={<Form />} />
       </Routes>
     </Router>
```

Comment:
- [ ] You can't use a Router inside another Router, kindly change this Router in line 12 and line 19 to <BrowserRouter> instead, because of this missed configuration I can't see the page, therefore, I can't review your page styles